deprecation-policy.md

```diff
@@ -84,3 +87,17 @@ Commands and their flags are subject to the deprecation policy based on the foll
 - init:  alpha 
 - run:  beta 
 - version:  beta 
+
+
+## Current deprecation notices 
+
+
+03/15/2019: With release v0.25.0 we mark for deprecation the `additionalFlags` field in kaniko, instead the `buildArgs` field should be used.
```

Comment:
This doesn't account for additional flags that aren't `--build-arg`. Maybe we could add in a sentence like:

The buildArgs field should now be used for build arguments, any additional flags kaniko accepts should be added as fields in the new KanikoArtifact type.